### ISSUE TITLE
Add support for Heroku-20

### ISSUE THREAD

**Issue by edmorley** [mars/create-react-app-buildpack#179]
Hi! Since this is a popular buildpack, it would be great to add support for the upcoming Heroku-20 stack.

The Nodejs buildpack already supports Heroku-20, so I believe the only piece left is [heroku-buildpack-static](https://github.com/heroku/heroku-buildpack-static), for which there is this ticket open:
https://github.com/heroku/heroku-buildpack-static/issues/166

Fixing this will mean also needing to fix:
https://github.com/heroku/heroku-buildpack-static/issues/165

**Comment by edmorley** [comment]
Heroku-20 is now in beta:
https://devcenter.heroku.com/changelog-items/1937

**Comment by edmorley** [comment]
Fixed by heroku/heroku-buildpack-static/pull/182.

Example CRA build:

```
$ npx create-react-app cra-test && cd $_
...
$ h create --stack heroku-20 --buildpack https://github.com/mars/create-react-app-buildpack
...
$ git push heroku main
...
remote: Building source:
remote:
remote: -----> React.js (create-react-app) multi app detected
remote: =====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-multi.git
remote: =====> Detected Framework: Multipack
remote: =====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-nodejs.git
remote: =====> Detected Framework: Node.js
...
remote: =====> Downloading Buildpack: https://github.com/mars/create-react-app-inner-buildpack.git#v9.0.0
remote: =====> Detected Framework: React.js (create-react-app)
remote:        Writing `static.json` to support create-react-app
remote:        Enabling runtime environment variables
remote: =====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-static.git
remote: =====> Detected Framework: Static HTML
remote: -----> Installed nginx 1.19.0 to /app/bin
remote: Using release configuration from last framework (Static HTML).
remote: -----> Discovering process types
remote:        Procfile declares types     -> (none)
remote:        Default types for buildpack -> web
remote:
remote: -----> Compressing...
remote:        Done: 63M
remote: -----> Launching...
remote:        Released v3
```
